libraries/AP_HAL_ChibiOS/hwdef/CubeOrange-periph-heavy/hwdef.dat

```diff
@@ -11,9 +11,6 @@ APJ_BOARD_ID 1400
 
 # setup build for a peripheral firmware
 env AP_PERIPH 1
-env AP_PERIPH_HEAVY 1
```

Comment:
@magicrub it seems you are the only one that wants heavy cube orange? Why do you want it? what features do you want?

Reply:
Over the years, it's been a hack to help (greatly) in getting libraries to build without having to hack the whole codebase to get things to build. I don't think I have any convincing arguments to keep it these days. My current massive Periph device, the MAIM payload manager, isn't using it so I think it's safe to remove at this point. AP_Periph has successfully infected the ArduPilot library codebase!